app/src/main/java/at/aau/edu/wizards/ui/lobby/LobbyViewModel.kt

```diff
@@ -30,17 +30,26 @@ class LobbyViewModel(
         server.stopBroadcasting()
     }
 
+    var numplayer = 1
```

Comment:
Just from naming convention `numPlayer` and `checkTooManyPlayer`, nothing crucial

Reply:
completed